changelogs/changelog.yaml

```diff
@@ -5,154 +5,154 @@ releases:
   1.1.0:
```

Comment:
Isn't this going to have the same issue as in the cloud.terraform collection?

Reply:
Do you mean the fact that this file is generated?

Reply:
Yeah

Reply:
updated